server/pages/index.jade

```diff
@@ -133,5 +133,23 @@ html(lang=lang, dir=isRTL ? 'rtl' : 'ltr', class=isFluidWidth ? 'is-fluid-width'
 				script(src=urls[ chunk + '-min' ])
 		script(type='text/javascript')!='window.AppBoot();'
 
+		script.
+			try {
+				if (window.console) {
+
+					if(true) {
+                        //Browser supports color output
+                        console.log("%cWait! This area is meant for developers!", "color: red;font-size: xx-large");
+					}else{
+						//No color supported
+                        console.log("Wait! This area is meant for developers!");
+					}
+                    console.log("If someone told you to type or paste something into this area it could compromise the security of your account.");
+
+                }
+			} catch (exception) {
+					// Do something with error
+            }
```

Comment:
why the `try/catch` if we are also checking for the presence of `window.console`? seems like we would only need one or the other